app/controllers/application_controller.rb

```diff
@@ -1,14 +1,22 @@
 class ApplicationController < ActionController::API
+  rescue_from JWT::DecodeError, with: :error_invalid
 
   def secret_key
-    "drinkthis"
+    ENV['GOOGLE_CLIENT_SECRET']
   end
 
   def encode(payload)
     JWT.encode(payload, secret_key, 'HS256')
   end
 
   def decode(token)
-    JWT.decode(token, "drinkthis", true, {algorithm: "HS256"})[0]
+    # body = JWT.decode(token, secret_key, true, {algorithm: "HS256"})[0]
+    # HashWithIndifferentAccess.new(body)
```

Comment:
Were these left in intentionally? And if so, could someone explain what's up?

Reply:
yes, they have been removed now.